src/Sulu/Bundle/MediaBundle/Media/Manager/DefaultMediaManager.php

```diff
@@ -823,7 +823,7 @@ public function increaseDownloadCounter($fileVersionId)
      * @param Media $media
      * @return Media
      */
-    protected function addFormatsAndUrl(Media $media)
+    public function addFormatsAndUrl(Media $media)
```

Comment:
if you want to use it anywhere in the system it should be added to the interface .... /cc @alexander-schranz is that a good idea to make that public?